build/org.eclipse.pde.build/src/org/eclipse/pde/internal/build/AbstractScriptGenerator.java

```diff
@@ -84,7 +85,7 @@ public abstract class AbstractScriptGenerator implements IXMLConstants, IPDEBuil
 	private static PDEUIStateWrapper pdeUIState;
 
 	/** Location of the plug-ins and fragments. */
-	protected String[] sitePaths;
+	protected List<File> sitePaths;
```

Comment:
Even though its already merged, why not even `use` Path instead of `File` ?

Reply:
I thought about that as well, but it would have required more changes. Since my main goal was to get rid of the File-to-URL and URL-to-File conversion I didn't want to do that as well in this change.
But since this is all internal we can change to Path at our own discretion at any time.